scripts/start.js

```diff
@@ -62,11 +62,9 @@ browserSync.init({
             match: [
                 `${CONFIG.sourceDir}/*.js`,
                 `${CONFIG.sourceDir}/**/*.js`,
-                // OLD Shadow DOM
-                `${CONFIG.sourceDir}/elements/*.less`, // (+) ShadowDOM CSS
-                `${CONFIG.sourceDir}/elements/*.html`, // (+) ShadowDOM Markup
-                // NEW Shadow DOM
-                `${CONFIG.sourceDir}/elements/**/_shadow.*`,
+                `${CONFIG.sourceDir}/elements/**/*`,
+                // ignore test files
+                `!${CONFIG.sourceDir}/**/*.spec.js`,
```

Comment:
Updated the globs to compile JavaScript when _any_ file in the `src/elements/` directory changes. This is because everything in that directory has a direct affect on compiled JavaScript (with the exception of spec files).